bundles/org.openhab.binding.unifi/src/main/resources/OH-INF/update/thing-updates.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
+<update:update-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:update="https://openhab.org/schemas/update-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/update-description/v1.0.0 https://openhab.org/schemas/update-description-1.0.0.xsd">
+
+	<thing-type uid="unifi:accessPoint">
+		<instruction-set targetVersion="1">
+			<add-channel id="online">
+				<type>unifi:online</type>
+				<description>Online status of the device</description>
+			</add-channel>
+			<add-channel id="state">
+				<type>unifi:devState</type>
+				<label>Device State</label>
```

Comment:
The label is not needed here, since it's not overridden on channel level, so it will be obtained from the channel type.
```suggestion
```

Reply:
Removed